docs/streams/developer-guide/kafka-streams-group-sh.html

```diff
@@ -0,0 +1,278 @@
+
+                <li><strong>Describe a Streams group</strong> and show:
+                    <ul>
+                        <li>Group state, group epoch, target assignment epoch (with <code>--state</code>, <code>--verbose</code> for additional details).</li>
+                        <li>Per‑member info such as epochs, current vs target assignments, and whether a member still uses the classic protocol (with <code>--members</code> and <code>--verbose</code>).</li>
+                        <li>Input‑topic offsets and lag (with <code>--offsets</code>), to understand how far behind processing is.</li>
+                    </ul>
+                </li>
+
+                <li><strong>Reset input‑topic offsets</strong> for a Streams group to control reprocessing boundaries using precise specifiers (earliest, latest, to‑offset, to‑datetime, by‑duration, shift‑by, from‑file). Requires <code>--dry-run</code> or <code>--execute</code> and inactive instances.</li>
+
+                <li><strong>Delete offsets</strong> for input topics to force re‑consumption on next start.</li>
+
+                <li><strong>Delete a Streams group</strong> to clean up broker‑side Streams metadata (offsets, topology, assignments). Optionally delete a subset of <strong>internal topics</strong> at the same time using <code>--internal-topics</code>.</li>
+            </ul>
+        </div>
+
+        <div id="usage">
+            <span class="anchor" aria-hidden="true"></span>
+            <h2>Usage</h2>
+            <p>The script is typically located in <code>bin/kafka-streams-groups.sh</code> and connects to your cluster via <code>--bootstrap-server</code>. For secured clusters, pass AdminClient properties using <code>--command-config</code>.</p>
+
+            <pre><code>$ kafka-streams-groups.sh --bootstrap-server &lt;host:port&gt; [COMMAND] [OPTIONS]</code></pre>
+
+            <div class="admonition note">
+                <strong>Note:</strong> <code>kafka-streams-groups.sh</code> complements the Streams Admin API for Streams groups. The CLI exposes list/describe/delete operations and offset management similar in spirit to consumer-group tools, but tailored to Streams groups defined in KIP‑1071.
+            </div>
+        </div>
+
+        <div id="commands">
+            <span class="anchor" aria-hidden="true"></span>
+            <h2>Commands</h2>
+
+            <div id="list">
+                <h3>List Streams groups</h3>
+                <p>Discovering groups</p>
+                <pre><code># List all Streams groups
+kafka-streams-groups.sh --bootstrap-server localhost:9092 --list
+</code></pre>
+            </div>
+
+            <div id="describe">
+                <h3>Describe Streams groups</h3>
+                <p>Inspecting group's state, members, and lag</p>
+                <pre><code>
+# Describe a group: state + epochs
```

Comment:
Should this to got the line about, to avoid an empty first line in the "code" block?